docs/install.rst

```diff
@@ -34,6 +34,19 @@ To do so, use the ``simbad-database`` command with the ``lattice`` subcommand. I
 
 Hit the ``<Enter>`` key and your default database will be updated automatically.
 
+If your CCP4 installation is located in a write protected directory, you can use the following command:
+
+.. code-block:: bash
+
+   $ sudo -E simbad-database lattice
+
+Alternativelym if this doesn't work, you can write the database to your working directory and then move it to the CCP4 installation using the following commands:
```

Comment:
typo after Alternatively

Reply:
"you can write the database to any user-specific directory and the move it across to the CCP4 installation ...."